alert-database/src/main/java/com/synopsys/integration/alert/database/job/blackduck/BlackDuckJobDetailsEntity.java

```diff
@@ -0,0 +1,119 @@
+package com.synopsys.integration.alert.database.job.blackduck;
+
+import java.util.List;
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import com.synopsys.integration.alert.database.job.blackduck.notification.BlackDuckJobNotificationTypeEntity;
+import com.synopsys.integration.alert.database.job.blackduck.policy.BlackDuckJobPolicyFilterEntity;
+import com.synopsys.integration.alert.database.job.blackduck.projects.BlackDuckJobProjectEntity;
+import com.synopsys.integration.alert.database.job.blackduck.vulnerability.BlackDuckJobVulnerabilitySeverityFilterEntity;
+
+@Entity
+@Table(schema = "alert", name = "blackduck_job_details")
+public class BlackDuckJobDetailsEntity {
+    @Id
+    @Column(name = "job_id")
+    private UUID jobId;
+
+    @Column(name = "global_config_id")
+    private Long globalConfigId;
+
+    @Column(name = "filter_by_project")
+    private Boolean filterByProject;
+
+    @Column(name = "project_name_pattern")
+    private String projectNamePattern;
+
+    @OneToMany(mappedBy = "blackDuckJobDetails")
```

Comment:
Should we have this be blackduck instead of blackDuck? There are more instances if we need to change this.

Reply:
ugh lol, not sure

Reply:
I think for capitalized usages and camelCase, we want it to be BlackDuck and blackDuck respectively. With snake case, all characters should be lowercase, but it should still be one word.